core/templates/pages/collection-editor-page/editor-tab/collection-node-editor.component.html

```diff
@@ -1,27 +1,36 @@
-<md-card class="collection-editor-node">
+<mat-card class="collection-editor-node">
   <div class="collection-editor-node-title">
-    <a ng-href="/create/<[$ctrl.getCollectionNode().getExplorationId()]>" target="_blank" rel="noopener">
-      <[$ctrl.getCollectionNode().getExplorationTitle()]>
+    <a [href]="hrefUrl"
+       target="_blank"
+       rel="noopener">
+      {{ collectionNode.getExplorationTitle() }}
     </a>
   </div>
   <span class="collection-editor-delete-node protractor-test-editor-delete-node">
-    <span ng-click="$ctrl.deleteNode()" aria-hidden="true">
-      <i class="material-icons md-18">&#xE14C;</i>
+    <span (click)="deleteNode()"
+          aria-hidden="true">
+      <i class="material-icons mat-18">
+        &#xE14C;
+      </i>
     </span>
   </span>
-  <span ng-click="$ctrl.shiftNodeLeft()"
-        ng-if="$ctrl.getLinearIndex() > 0"
+  <span (click)="shiftNodeLeft()"
+        *ngIf="linearIndex > 0"
         class="collection-editor-shift-left-arrow protractor-test-editor-shift-left"
         aria-hidden="true">
-    <i class="material-icons">&#xE5CB;</i>
+    <i class="material-icons">
+      &#xE5CB;
+    </i>
   </span>
-  <span ng-click="$ctrl.shiftNodeRight()"
-        ng-if="$ctrl.getLinearIndex() < $ctrl.collection.getCollectionNodeCount() - 1"
+  <span (click)="shiftNodeRight()"
+        *ngIf="linearIndex < collection?.getCollectionNodeCount() - 1"
```

Comment:
why `?`

Reply:
HTML here tries to check this condition before our ngOnInit gets executed. So to ensure that collection are fetched first `?` is required.Without this change it gives `cannot read property getCollectionNodeCount of undefined`